django/shortcuts.py

```diff
@@ -51,8 +51,10 @@ def redirect(to, *args, permanent=False, **kwargs):
 
         * A URL, which will be used as-is for the redirect location.
 
+    URLs are automatically encoded.
```

Comment:
This is only the case if a pattern name is passed in, and the result of `reverse()` is returned, other urls are returned as-is.